app/views/foreman/unattended/finish-katello.erb

```diff
@@ -60,6 +60,8 @@ cat > /etc/puppet/puppet.conf << EOF
 <%= snippet 'puppet.conf' %>
 EOF
 
+<%= snippet('remote_execution_ssh_keys') %>
```

Comment:
Does this snippet exist if remote execution is not installed? Will it error if not or gracefully ignore?